lib/src/flex_with_main_child.dart

```diff
@@ -39,7 +39,7 @@ class _FlexWithMainChildState extends State<FlexWithMainChild> {
 
   @override
   Widget build(BuildContext context) {
-    WidgetsBinding.instance!.addPostFrameCallback((_) {
```

Comment:
Although this would fix the issue with flutter 3, it would make the package incompatible with flutter 2. I'd much prefer something that works for both versions. There's a fix for another package that might also work here: https://github.com/CoderUni/responsive_sizer/commit/e6bb1643b8769c951eeeac5d7dc8d198fc5ce6a6#diff-0730f7244987f77976094a672f55f4401ca3cd433c91e8dcfe8ab7c1525fa620